app/src/main/res/values/strings.xml

```diff
@@ -24,6 +24,7 @@
     <string name="submit">Submit</string>
     <string name="type">Type</string>
     <string name="yes">Yes</string>
+    <string name="dash">-</string>
```

Comment:
`name="dash" translatable="false"`
Then you can remove dash from strings-nl.xml